client/app/reportAnimal/components/ReportAnimal.js

```diff
@@ -711,4 +711,4 @@ const ReportAnimal = () => {
 }
 
 
-export default ReportAnimal
```

Comment:
ignore this this is the file for my next PR

Reply:
🫡